packages/paste-core/components/tabs/src/Tab.tsx

```diff
@@ -128,6 +128,7 @@ const Tab = React.forwardRef<HTMLDivElement, TabProps>(({children, ...tabProps},
               {...textStyles}
               fontSize="fontSize30"
               fontWeight="fontWeightSemibold"
+              lineHeight="lineHeight20"
```

Comment:
Can you remove this? The 20px line height will come by default when we merge in https://github.com/twilio-labs/paste/pull/656.

That way we keep the line heights in components in line with their font sizes